backend/Dockerfile

```diff
@@ -55,6 +57,8 @@ FROM debian:bookworm-slim
 WORKDIR /app
 
 COPY --chmod=755 --from=build-prod /app/target/release/arcadia-index .
+COPY --from=prebuild /usr/local/cargo/bin/sqlx .
+COPY  ./migrations ./migrations
```

Comment:
I took this approach ensures the schema is in lockstep with what the binary expects - I believe mounting it in the docker compose would only open the door to drift